docs/index.md

```diff
@@ -0,0 +1,3 @@
+{% for version in site.versions %}
```

Comment:
This wouldn't be required in `master` branch. It is useful in `gh-pages` actually.